src/components/datetimepicker/Datetimepicker.vue

```diff
@@ -145,6 +146,10 @@ export default {
         },
         minDatetime: Date,
         maxDatetime: Date,
+        nearbyMonthDays: {
+            type: Boolean,
+            default: true
```

Comment:
To respect the default behavior of `Datepicker`, I think the following would be better:
```js
default: () => config.defaultDatepickerNearbyMonthDays
```
Or leave it `undefined` by default.